cppcache/integration-test/testThinClientSecurityAuthentication.cpp

```diff
@@ -138,7 +140,7 @@ DUNIT_TASK_DEFINITION(LOCATORSERVER, CreateServer1)
         printf("Input to server cmd is -->  %s",
                cmdServerAuthenticator.c_str());
         CacheHelper::initServer(
-            1, nullptr, locHostPort,
+            1, "cacheserver_notify_subscription.xml", locHostPort,
```

Comment:
Wouldn't this change the intent of the test now that it starts with a cache xml file?

Reply:
N/A. Test deleted.